openslide/__init__.py

```diff
@@ -27,7 +27,7 @@
 from PIL import Image
 
 from openslide import lowlevel
-from openslide._version import __version__
+from openslide._version import __version__  # noqa: F401
```

Comment:
Could you add comments at the places where you annotate with noqa? Especially in the cases where it uses an opaque code like this.

Reply:
I can write out the descriptive text of the code, for instance this one is module-imported-but-unused, which is mostly because it is only imported to expose as openslide.__version__ in the API.

An alternative way is to use __all__ to list all re-exported symbols which would 'use' the import and the flake8 warning would go away as well.

AFAIK, existing comments addressed why the "unnecessary" imports were done and I added a comment indicating where the undefined __version__ symbol in setup.py comes from.